packages/frontend/src/components/campaign-details/index.tsx

```diff
@@ -1,28 +1,32 @@
 "use client";
 
 import type { SupportedChain } from "@metrom-xyz/contracts";
+import { useCampaign } from "@/src/hooks/useCampaign";
+import type { Address } from "viem";
 import { useRouter } from "@/src/navigation";
 import { Nav } from "../nav";
+import { Header, SkeletonHeader } from "./header";
 
 import styles from "./styles.module.css";
 
 interface CampaignDetailsProps {
     chain: SupportedChain;
-    campaignId: string;
+    campaignId: Address;
```

Comment:
Same as above

Reply:
Fixed in 328249fe1c2965f43e5177cfc2abbaf5b371501d.